virtcontainers/clh.go

```diff
@@ -128,9 +128,9 @@ type cloudHypervisor struct {
 var clhKernelParams = []Param{
 
 	{"root", "/dev/pmem0p1"},
-	{"panic", "1"},         // upon kernel panic wait 1 second before reboot
-	{"no_timer_check", ""}, // do not check broken timer IRQ resources
-	{"noreplace-smp", ""},  // do not replace SMP instructions
+	{"panic", "1"},                                     // upon kernel panic wait 1 second before reboot
+	{"no_timer_check", ""},                             // do not check broken timer IRQ resources
+	{"noreplace-smp", ""},                              // do not replace SMP instructions
```

Comment:
This is where `gofmt` is not happy about. Looks like your `gofmt` made the change?